test/manipulation.test.js

```diff
@@ -882,6 +882,9 @@ describe('manipulation', function() {
     });
 
     it('should allow save() of the created instance', function(done) {
+      Person.create(
+        {name: 'a-name', gender: undefined, id: 999});
+
```

Comment:
This is against the intent of the test as I understand that intent - call `updateOrCreate` in such way that a new instance is created via the connector method, then call `save` on that method.

I think you need a model with `forceId: false` to test that scenario.